src/main/scala/tech/cryptonomic/conseil/routes/openapi/DataJsonSchemas.scala

```diff
@@ -51,6 +56,9 @@ trait DataJsonSchemas extends algebra.JsonSchemas with generic.JsonSchemas   {
   /** Query response schema schema */
   implicit def queryResponseSchema: JsonSchema[List[QueryResponse]]
 
+  /** Query response schema schema */
+  implicit def queryResponseSchemaWithCsv: JsonSchema[QueryResponseWithOutput]
```

Comment:
maybe a better naming? 😁

Reply:
changed it to `queryResponseSchemaWithOutputType`, maybe you have other suggestions? 😄